libraries/utilities/generate_clusters_from_pool.py

```diff
@@ -156,17 +160,22 @@ def write_config(config, pool_file):
         try:
             f.write("[webhook_ip]\n")
             if platform.system() == "Darwin":
-                local_ip = socket.gethostbyname(socket.gethostname())
+                # HACK: http://stackoverflow.com/questions/166506/finding-local-ip-addresses-using-pythons-stdlib
```

Comment:
Glad to see the hack documented, but can you make it clear what's being done here?  "Find the IP address of the machine this code is running on on the LAN so that other machines on the LAN can connect to it" <-- ?